### PR TITLE
feat(composer): Support API Elements

### PR DESCRIPTION
**Pull request by kylef**

### CODE REVIEW

File: source/composer.apib [hunk 1]
```diff
@@ -32,6 +38,14 @@ API Blueprint as defined in its [specification](https://github.com/apiaryio/api-
 
 + Relation: compose
 
++ Request Compose API Blueprint from API Elements API (application/vnd.refract+json)
+
+        :[](fixtures/apib/normal.refract.json)
```
Comment (w-vi) [inline source/composer.apib]: This file does not exist, did you mean `fixtures/apib/normal.refract.parse-result.json`?

Comment (kylef) [inline source/composer.apib]: Updated, forgot to `git add` the new file.

Comment (kylef) [inline source/composer.apib]: This file is similar to `fixtures/apib/normal.refract.parse-result.json` but it is not wrapped in a parse result.

Comment (w-vi) [inline source/composer.apib]: IC

**Comment by w-vi** [comment]
Feel free to merge once the Ci is done.

**Comment by kylef** [comment]
CI fails due to dredd testing, I don't think we should do this -> https://github.com/apiaryio/api.apiblueprint.org/pull/15

File: source/composer.apib [hunk 2]
```diff
@@ -6,6 +6,12 @@ The composition of API Blueprint is performed as it is provided by the [Matter C
 
 #### Input Media Types
 
+##### API Elements
+
+API Elements is supported, you can send an [API Elements API
+Category](http://api-elements.readthedocs.io/en/latest/element-definitions/#category-base-api-element)
+to the composer using the `application/vnd.refract+json` content type.
```
Comment (pksunkara) [inline source/composer.apib]: I think parse result should be the input.

Comment (pksunkara) [inline source/composer.apib]: Because it is as a whole is "API Elements"

Comment (pksunkara) [inline source/composer.apib]: We have decided to support both Parse Result and API Category. So, let's edit this description and have both transaction examples here.

File: source/composer.apib [hunk 3]
```diff
@@ -6,6 +6,20 @@ The composition of API Blueprint is performed as it is provided by the [Matter C
...
+API Elements is supported, you can send an [API Elements API
+Category](http://api-elements.readthedocs.io/en/latest/element-definitions/#category-base-api-element)
+to the composer using the `application/vnd.refract+json` content type. You may
+also send an [API Elements Parse
+Result](http://api-elements.readthedocs.io/en/latest/element-definitions/#parse-result-elements)
+as `application/vnd.refract.parse-result+json`.
+
+```
+application/vnd.refract+json
+application/vnd.refract.parse-result+json
```
Comment (pksunkara) [inline source/composer.apib]: You also need to add a transaction example for the parse result too. We are using this APIB to do integration testing.

Comment (kylef) [inline source/composer.apib]: Updated https://github.com/apiaryio/api.apiblueprint.org/pull/14/commits/8d7891309eea11cd94a8f6c477b094531c44dd7a

File: source/fixtures/apib/composer.apib [hunk 4]
```diff
@@ -0,0 +1,9 @@
+FORMAT: 1A
+
+# Hello API
+
+## Group 
+
+### /message
+
+#### GET
```
Comment (pksunkara) [inline source/fixtures/apib/composer.apib]: @kylef What happened to the response here?

**Pull request merged**